test-cases/longevity/longevity-cdc-3d-400gb.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 300
+
+prepare_write_cmd: ["cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb.yaml ops'(insert=1)' cl=QUORUM n=30000000 -port jmx=6868 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage.yaml ops'(insert=1)' cl=QUORUM n=30000000 -port jmx=6868 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_postimage.yaml ops'(insert=1)' cl=QUORUM n=30000000 -port jmx=6868 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage_postimage.yaml ops'(insert=1)' cl=QUORUM n=30000000 -port jmx=6868 -mode cql3 native -rate threads=100"
+                   ]
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=240m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=240m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=240m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=240m -port jmx=6868 -mode cql3 native -rate threads=100"
+             ]
+
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 5
+nemesis_during_prepare: false
+
+user_prefix: 'longevity-cdc-3d-400gb'
+space_node_threshold: 322122547200
```

Comment:
How do you calculate this number?

Reply:
300 * 1024^3

Reply:
Why do you want the nemesis to start only after 300gb?

Reply:
removed